src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -45,6 +48,50 @@ def _set_uniq_aggregation_on_session_status_factory(
     )
 
 
+def _aggregation_on_tx_status_func_factory(aggregate):
+    def _get_query_functions(org_id, metric_ids, exclude_tx_statuses):
+        metric_match = Function("in", [Column("metric_id"), list(metric_ids)])
+        if len(exclude_tx_statuses) == 0:
```

Comment:
```suggestion
        assert exclude_tx_statuses is not None
        if len(exclude_tx_statuses) == 0:
```

Reply:
I don't understand this suggestion. `exclude_tx_statuses` is a `List`, and the behavior depends on whether the list is empty; I don't know how you're thinking of achieving that with an assertion.

Reply:
There is nothing that prevents against sending `exclude_tx_statuses=None` here. You can't call `len` on a `None` object

Reply:
Oh, I read the suggestion wrong (the diff seems to suggest the removal of the `if`). Good suggestion, updated.